docs/pages/app-signing/app-credentials.mdx

```diff
@@ -129,4 +129,4 @@ Running the command will ask you to select a build that you want to re-sign. For
 
 After selecting the build, follow the steps to log in to your Apple Developer account. When prompted **Show devices and ask me again**, you can select a new provisioning profile.
 
-Select a new device, and the command will run the EAS Build again. Note that the build triggered this time reuses the application artifact from the selected build and codesigns it with the new provisioning profile. Once the process of codesigning is complete, you can use this new build link to install the **.ipa** on the iOS device that was added to the provisioning profile during the process.
+Select a new device, and the command will run the EAS Build again. Note that the build triggered this time reuses the application artifact from the selected build and resigns it with the new provisioning profile. Once the process of resigning is complete, you can use this new build link to install the **.ipa** on the iOS device that was added to the provisioning profile during the process.
```

Comment:
```suggestion
Select a new device, and the command will run the EAS Build again. Note that the build triggered this time reuses the application artifact from the selected build and codesigns it with the new provisioning profile. Once this process is complete, you can use this new build link to install the **.ipa** on the iOS device added to the provisioning profile.
```

I think the confusing part here is the last line but again, in the second line, the process of code signing an ipa is being described so let's stick to the terminology.